content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -2,7 +2,47 @@
 name: Workspace
 match: [workspace]
 tooltip: >-
-  Directory containing all your project files e.g. raw datasets, source code, ML
-  models, etc. Typically, it's also a Git repository. It will contain your DVC
+  The directory containing all your project files, e.g., the raw data, source
+  code, ML models. Typically, it's also a Git repository. It contains your DVC
   project.
 ---
+
+# Workspace
+
+A data science project can consist of data obtained from many distinct sources.
+These may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements, e.g., a
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly difficult.
+
+DVC allows a single directory to contain all your project artifacts. The
```

Comment:
> a single directory to contain all your project artifacts

Not exactly. File contents are in the cache with a special file structure (see https://dvc.org/doc/user-guide/project-structure/internal-files#structure-of-the-cache-directory)

> The workspace is the directory containing the _visible_ part of your project

That part is correct. And contradicts the previous part 🙂 (because "visible part" implies there's a hidden part which must be in other dirs).

Reply:
> The workspace is the directory containing the visible part of your project

Let's open this p with that sentence.